sustc-api/src/main/java/io/sustc/service/UserService.java

```diff
@@ -27,19 +27,19 @@ public interface UserService {
      * Deletes a user.
      *
      * @param auth indicates the current user
-     * @param mid  the user to be deleted
+     * @param mid  the user's {@code mid} to be deleted
      * @return operation success or not
      * @apiNote You may consider the following corner cases:
      * <ul>
-     *   <li>{@code mid} is invalid (<= 0)</li>
+     *   <li>{@code mid} is invalid (<= 0 or do not exist)</li>
```

Comment:
```suggestion
     *   <li>cannot find a user corresponding to the {@code mid}</li>
```

mid <= 0 is included in this case